### PR TITLE
[TASK] Add section on core upgrader

### PR DESCRIPTION
**Pull request by garvinhicking**
References #273

I'm a bit unsure if we really want to add this due to the forking/maintenance nature, but on the other hand it could be helpful.

### CODE REVIEW

**Review by linawolf — APPROVED** [review]

File: Documentation/Tools/Index.rst [hunk 1]
```diff
@@ -30,3 +30,18 @@ Support
...
 channel. You can also open an issue or start a discussion on the projects GitHub page.
+
+EXT: Core Upgrader (v2)
+=======================
+
+The TYPO3 extension was initially developed as
+`EXT:core-upgrader <https://github.com/IchHabRecht/core_upgrader>` (Composer package
+`ichhabrecht/core-upgrader`, compatible up to TYPO3 v10) and has been forked as
+`EXT:core-upgraderv2 <https://github.com/WapplerSystems/core_upgrader>` (Composer package
+`wapplersystems/core-upgrader`, compatible up to TYPO3 v12) with).
```
Comment (froemken) [inline Documentation/Tools/Index.rst:41]: `with)` is too much

**Review by froemken — CHANGES_REQUESTED** [review]

File: Documentation/Tools/Index.rst (hunk 1 above)
Comment (garvinhicking) [inline Documentation/Tools/Index.rst:41]:
```suggestion
`wapplersystems/core-upgrader`, compatible up to TYPO3 v12).
```

Comment (garvinhicking) [reply Documentation/Tools/Index.rst]: thanks!

**Review by froemken — APPROVED** [review]
We at jweiland.net also work with either old core-upgrader or new core-upgrader. These extensions are only a collection of all original TYPO3 Core Upgrade classes. So, for me it's not a problem to mention these community extensions.

File: Documentation/Tools/Index.rst [hunk 2]
```diff
@@ -30,3 +30,18 @@ Support
...
+EXT: Core Upgrader (v2)
+=======================
+
+The TYPO3 extension was initially developed as
+`EXT:core-upgrader <https://github.com/IchHabRecht/core_upgrader>` (Composer package
+`ichhabrecht/core-upgrader`, compatible up to TYPO3 v10) and has been forked as
+`EXT:core-upgraderv2 <https://github.com/WapplerSystems/core_upgrader>` (Composer package
+`wapplersystems/core-upgrader`, compatible up to TYPO3 v12).
+
+The extension allows to perform multiple TYPO3 core version upgrades in one step by offering
```
Comment (brotkrueml) [inline Documentation/Tools/Index.rst:43]:
```suggestion
The extension allows to perform multiple TYPO3 Core version upgrades in one step by offering
```

**Pull request merged**